### PR TITLE
OCPBUGS-3784: UPSTREAM: 115210: Trim out suffix added to backing file path of loopback device

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

File: pkg/volume/util/volumepathhandler/volume_path_handler_linux_test.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+package volumepathhandler
+
+import (
+	"fmt"
+	"testing"
+)
+
+func pathWithSuffix(suffix string) string {
+	return fmt.Sprintf("%s%s", "/var/lib/kubelet/plugins/kubernetes.io/csi/volumeDevices/pvc-1d205234-06cd-4fe4-a7ea-0e8f3e2faf5f/dev/e196ebd3-2ab1-4185-bed4-b997ba38d1dc", suffix)
```
Comment (mpatlasov) [inline pkg/volume/util/volumepathhandler/volume_path_handler_linux_test.go:25]: This is unimportant bit, but the test would look cleaner if we factor out that long `/var/...dc` string to a global constant. This way we would avoid code-duplication with `const defaultPath = ...` below.

Comment (bertinatto) [reply pkg/volume/util/volumepathhandler/volume_path_handler_linux_test.go:25]: I'd be cleaner, indeed. I guess we're way more careless with unit tests. Will keep it mind next time, thanks!

**Comment by bertinatto** [comment]
This has been pulled in with the rebase.
/close

**Pull request closed without merging**